site/docs/try/authentication.md

```diff
@@ -0,0 +1,25 @@
+# Authentication
+
+By default, Nessie servers run with authentication disabled and all requests are processed under the "anonymous"
+user identity.
+
+## OpenID Bearer Tokens
+
+Nessie supports bearer tokens and uses [OpenID Connect](https://openid.net/connect/) for validating them.
+
+To enable bearer authentication the following [configuration](./configuration.md) properties need to be set 
+for the Nessie Server process:
+
+* `nessie.server.authentication.enabled=true`
+* `quarkus.oidc.auth-server-url=<OpenID Server URL>`
+* `quarkus.oidc.client-id=<Client ID>`
+
+When using Nessie [docker](./docker.md) images, the authentication options can be specified on
```

Comment:
Typo: 
```suggestion
When using Nessie [Docker](./docker.md) images, the authentication options can be specified on
```

Reply:
Fixed in 3b95a50929b5664beab05ed75e0380a42d703a6a